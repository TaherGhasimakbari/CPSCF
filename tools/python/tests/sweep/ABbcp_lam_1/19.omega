 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3538955114E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  1.035000000424E+01  4.650000006763E+00       0     1
 -3.526410042181E+00  1.487331589279E+00       1     2
 -4.772859789043E-01  5.695607038274E-01       2     2
  1.408042840152E-01  2.219146296388E-01       3     2
  6.238837500341E-02  3.930058180222E-02       4     2
  6.714079902405E-03 -3.128176223882E-03       5     2
 -2.460294686511E-03 -3.971793404137E-03       6     2
 -1.163383896960E-03 -1.098379144323E-03       7     2
 -1.839534523046E-04 -9.523193131530E-05       8     2
  1.677264928871E-05  3.721088495014E-05       9     2
  1.534014834428E-05  1.655098904934E-05      10     2
  3.379116306288E-06  2.724037648133E-06      11     2
  1.209781250095E-07 -1.131473552245E-07      12     2
 -1.410921486330E-07 -1.723843837510E-07      13     2
 -4.475975076048E-08 -4.176199308245E-08      14     2
 -4.864471088476E-09 -2.466296676771E-09      15     2
  1.072083746340E-09  1.782999354270E-09      16     1
