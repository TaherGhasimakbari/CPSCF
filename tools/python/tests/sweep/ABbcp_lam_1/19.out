 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.1000000022E-01    6.8999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3538955114E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.1951481866E+00
f_homo              
    2.2085000036E+00
pressure            
    3.7711549335E+00
mu_chain            
    5.9663031201E+00
stress              
    8.6933691398E-12

STATISTICS          
N_star              
                  17
Final Error         
    8.6933691398E-10
Iterations          
                  15
Basis Time          
    2.6167106628E-01
SCF Time            
    1.3542199135E-01
